text/chapter6.md

```diff
@@ -204,6 +204,7 @@ Again, strings and arrays are simple examples of monoids.
 A `Monoid` type class instance for a type describes how to _accumulate_ a result with that type, by starting with an "empty" value, and combining new results. For example, we can write a function which concatenates an array of values in some monoid by using a fold. In PSCi:
 
 ```haskell
+> import Prelude
```

Comment:
not sure this is needed

Reply:
Yes, I guess it depends if you run all examples in PSCi in succession or in isolation.

If this example is run in isolation then you'll need to import `append` from Prelude.

https://pursuit.purescript.org/packages/purescript-prelude/4.1.1/docs/Data.Semigroup#v:append